crates/solvers/src/infra/config/dex/file.rs

```diff
@@ -25,6 +25,10 @@ struct Config {
     #[serde_as(as = "Option<serde_with::DisplayFromStr>")]
     absolute_slippage: Option<eth::U256>,
 
+    /// The number of concurrent requests to make to the DEX aggregator API.
+    #[serde(default = "default_concurrent_requests")]
+    concurrent_requests: usize,
```

Comment:
Would it be possible to directly deserialize `NonZeroUsize` here?

Reply:
It is!